apps/apps.h

```diff
@@ -72,6 +72,7 @@ BIO *dup_bio_in(int format);
 BIO *dup_bio_out(int format);
 BIO *dup_bio_err(int format);
 BIO *bio_open_owner(const char *filename, int format, int private);
+BIO *bio_open_owner_tmp(const char *filename, int format, int private);
```

Comment:
Are there any calls to `bio_open_owner` left?  If not, why not simply replace it with the new stuff instead of creating a separate function?

Reply:
Ahh, I didn't find that after replacing all the `bio_open_owner` to `bio_open_owner_tmp`, there is no `bio_open_owner` anymore... Going to fix this.